frontend/packages/ux-editor/src/containers/DesignView/DesignView.tsx

```diff
@@ -88,25 +87,29 @@ export const DesignView = (): ReactNode => {
     // If the layout does not exist, return null
     if (layout === undefined) return null;
 
+    // Check if the layout has unique component IDs
+    const isValidLayout = !duplicatedIdsExistsInLayout(layout.data);
     return (
-      <PageAccordion
-        pageName={layout.page}
-        key={i}
-        isOpen={layout.page === selectedFormLayoutName}
-        onClick={() => handleClickAccordion(layout.page)}
-      >
-        {layout.page === selectedFormLayoutName && <FormLayout layout={layout.data} />}
-      </PageAccordion>
+      <Accordion key={i} color={'neutral'}>
```

Comment:
Are you sure we want to have an Accordion for each element in the list? 
```PageAccordion``` is an ```<Accordion.Item/>``` component, and it should only be one <Accordion /> around a group of accordion items 😄 
Great that you add the color to it! However, I'll suggest dropping the ```{``` as it is only text 😄

Reply:
You are right, I'm unsure why I changed it to this. Reversed it back.